src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
However, `callSetOwnerFirst` already requires an Extensible object so the CustomMethod code already depends on Extensible being applied.